### PR TITLE
Spelling: differing sources and/or context

### PR DESCRIPTION
**Pull request by comradekingu**
## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [ ] I have squashed my commits into logic units.
- [ ] I have described the changes in the commit messages.

### CODE REVIEW

File: weblate/templates/translate.html [hunk 1]
```diff
@@ -318,31 +318,31 @@ <h4 class="panel-title">
...
+    <tr><th colspan="5" class="warning">{% blocktrans count count=others.source|length %}This string has a different context, but the same source.{% plural %}These strings have differing context, but the same source.{% endblocktrans %}</th></tr>
     <tbody>
     {% for item in others.source %}
     {% include "trans/other-row.html" %}
     {% endfor %}
     </tbody>
     {% endif %}
     {% if others.context %}
-    <tr><th colspan="5" class="warning">{% blocktrans count count=others.context|length %}The following string has a different source, but the same context.{% plural %}The following strings have different sources, but the same context.{% endblocktrans %}</th></tr>
+    <tr><th colspan="5" class="warning">{% blocktrans count count=others.context|length %}This string has a different source, but the same context.{% plural %}These strings have differing sources, but the same context.{% endblocktrans %}</th></tr>
```
Comment (comradekingu) [inline weblate/templates/translate.html:337]: ("differing" to make it clear it is different to the original string, as opposed to just all different between each-other.)

Comment (nijel) [reply weblate/templates/translate.html:337]: Shouldn't the singular have differing as well?

Comment (comradekingu) [reply weblate/templates/translate.html:337]: Not how I think of it. Might have to consult someone.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**